script.js

```diff
@@ -1,4 +1,193 @@
+//There are 2 players and players take turns.
+// When a player clicks Submit, the game rolls 2 dice and shows the dice rolls, for example 3 and 6.
+// The player picks the order of the dice they want. For example, if they wanted the number 63, they would specify that the 2nd dice goes first. You can choose how the player specifies dice order.
+// After both players have rolled and chosen dice order, the player with the higher combined number wins.
+
+//declaring variables to be used throughout
+var gameStep = 0;
+var player1Array = [1, 0];
+var player2Array = [2, 0];
+var playerIsNumber = 1;
+var myOutputValue = ``;
+var mainDiceRoll1;
+var mainDiceRoll2;
+// random number generator
+var randomDiceNumberGenerator = function () {
+  // generating a number from 0 to 5 both inclusive
+  var randomNumber = Math.floor(Math.random() * 6);
+  //making the number 1 to 6 both inclusive
+  var randomDiceNumber = randomNumber + 1;
+  return randomDiceNumber;
+};
+//generating dice roll
+var diceRoll = function () {
+  var roll1 = randomDiceNumberGenerator();
+  return roll1;
+};
```

Comment:
Nice that you want to make a function here, but this function is a little redundant 😅
perhaps you can directly rename randomDiceNumberGenerator as diceRoll ?